.github/workflows/build.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Setup OCaml ${{ matrix.ocaml-version }}
         uses: ocaml/setup-ocaml@v2
         with:
-          ocaml-version: ${{ matrix.ocaml-version }}
+          ocaml-compiler: ${{ matrix.ocaml-version }}
```

Comment:
I agree this workflow is useless and can be removed. The `esy` workflow should be kept though.